cluster-provision/k8s/provision.sh

```diff
@@ -4,7 +4,7 @@
 set -ex
 
 PHASES_DEFAULT="linux,k8s"
-PHASES="${PHASES:-$PHASES_DEFAULT}"
+PHASES="$PHASES_DEFAULT"
```

Comment:
please drop, not related it seems

Reply:
Yeah this is just for debugging, will drop once the PR is ready